cmd/kubeadm/app/cmd/util/cmdutil.go

```diff
@@ -45,16 +45,16 @@ import (
 func SubCmdRun() func(c *cobra.Command, args []string) {
 	return func(c *cobra.Command, args []string) {
 		if len(args) > 0 {
-			kubeadmutil.CheckErr(usageErrorf(c, "invalid subcommand %q", strings.Join(args, " ")))
+			kubeadmutil.CheckErr(usageErrorf(c, "%w %q", kubeadmutil.ErrInvalidSubCommand, strings.Join(args, " ")))
 		}
 		c.Help()
 		kubeadmutil.CheckErr(kubeadmutil.ErrExit)
 	}
 }
 
 func usageErrorf(c *cobra.Command, format string, args ...interface{}) error {
-	msg := fmt.Sprintf(format, args...)
-	return errors.Errorf("%s\nSee '%s -h' for help and examples", msg, c.CommandPath())
+	msg := fmt.Errorf(format, args...)
+	return errors.WithStack(fmt.Errorf("%w\nSee '%s -h' for help and examples", msg, c.CommandPath()))
```

Comment:
i'm not sure why we are using %w and then calling with WithStack.
note that we only care about stacktrace for --v=5.

general usage of %w is not present at all in kubeadm.